java/src/client/game/GameManager.java

```diff
@@ -158,6 +159,7 @@ public void setClientModel(ClientModel cm) {
     }
 
     public PlayerInfo getPlayerInfo() {
+        if (playerInfo == null) playerInfo = new PlayerInfo();
```

Comment:
Same as above, but also explicitly state the `@Nullable`. This would make sense if there is no current player e.g. when we haven't logged in.